lib/data_hygiene/policy_tagger.rb

```diff
@@ -0,0 +1,80 @@
+class PolicyTagger
+  def self.process_from_csv(csv_location, logger: Logger.new(nil))
+    CSV.foreach(csv_location, headers: true) do |row|
+      policies_to_remove = row.fetch("policies_to_remove") ? policies_to_remove.split(" ") : []
```

Comment:
`policies_to_remove` can be nil and therefore I won't be able to run `.split` on it. 
If a value for those is not provided means that the PolicyTagger should not do nothing on that regard. For example:

- `policies_to_remove` not provided: do not untag policies from Edition
- `policies_to_add` not provided: do not tag policies from Edition

I've now explained how I expect the CSV to look like in the description. I hope that makes things clear.